webapp/src/components/atoms/SpinButton/SpinButton.css

```diff
@@ -0,0 +1,62 @@
+.spin-button__box {
```

Comment:
I really like this naming convention (BEM? I think)

Didn't include it in this project but there's as an FYI: there's some stuff we can use to scope the CSS to within the react component which is really handy for changing all the style of a common element but only within a page.

I don't recall off hand what the plugins/config/library it was off-hand. `postcss` or something maybe?

Reply:
Yes it's BEM! I've only used SCSS to create reusable styling so I will definitely look into the library/postcss you mentioned!